### ISSUE TITLE
user parameter on mco deploy gets ignored

### ISSUE THREAD

**Issue by tampakrap** [acidprime/r10k#270]
I tried the command `mco r10k deploy production user=puppet`, but the user attr seems to get ignored. In the agent/r10k.rb I tried to print `request[:user]` but it gave me an empty string

**Comment by rnelson0** [comment]
@tampakrap Did you mean to open this on puppetlabs/r10k instead?

**Comment by tampakrap** [comment]
nope, the mcollective r10k plugin is provided from this module, see files/{agent,application}

**Comment by rnelson0** [comment]
Ah it looks like it's `agent/r10k.ddl` now, that's why I couldn't find it. Well see if we can get to this soon. Are you ever on IRC/slack to do some interactive testing?

**Comment by tampakrap** [comment]
I am on IRC in `#voxpupuli` but this issue is not affecting me any more, I kept it open just in case someone is facing it as well and would like to fix it :)
